rapids-cmake/cmake/support_conda_env.cmake

```diff
@@ -75,12 +84,31 @@ function(rapids_cmake_support_conda_env target)
     set(prefix_paths)
 
     if(in_conda_build)
+      # For conda-build we add the host conda environment prefix to the cmake search paths so that
+      # raw `find_file` or `find_library` calls will find CUDA components in the host environment
+      set(target_platform $ENV{cross_target_platform}) # when target != cross_target
+      if(NOT target_platform)
+        set(target_platform $ENV{target_platform})
+      endif()
+      if("${target_platform}" STREQUAL "linux-64")
+        set(targetsDir "targets/x86_64-linux")
+      elseif("${target_platform}" STREQUAL "linux-ppc64le")
+        set(targetsDir "targets/ppc64le-linux")
+      elseif("${target_platform}" STREQUAL "linux-aarch64")
+        set(targetsDir "targets/sbsa-linux")
+      endif()
+
       target_include_directories(${target} INTERFACE "$ENV{PREFIX}/include"
                                                      "$ENV{BUILD_PREFIX}/include")
       target_link_directories(${target} INTERFACE "$ENV{PREFIX}/lib" "$ENV{BUILD_PREFIX}/lib")
 
       if(DEFINED CMAKE_SHARED_LIBRARY_RPATH_LINK_CUDA_FLAG
          OR DEFINED CMAKE_SHARED_LIBRARY_RPATH_LINK_CXX_FLAG)
+        if(DEFINED targetsDir)
+          target_link_options(${target} INTERFACE
```

Comment:
Do we actually want to modify the rpath as well? I thought that the whole point of the structure we've laid out is to ensure that libraries don't rely on anything other than the proper SONAME of the package. If we include this directory we'll be silently allowing the usage of full library names.

Reply:
OK got it that works then. Just didn't want to add extra RPATH/RUN_PATH entries.